modules/core/src/main/scala/org/scalasteward/core/application/RunResults.scala

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2018-2023 Scala Steward contributors
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.scalasteward.core.application
+
+import cats.effect.ExitCode
+import cats.syntax.all._
+import org.scalasteward.core.data.Repo
+
+case class RunResults(results: List[Either[(Repo, Throwable), Repo]]) {
```

Comment:
**suggestion:** We could also create this class as a utility method (something like `GitHubSummary.from(results)`), since it is only used on one place for generating the summary.

<sub>This comment follows the [**conventionalcomments.org**](https://conventionalcomments.org) standard</sub>

Reply:
It does have two outputs (`exitCode`, as well as `markdownSummary`), so maybe it makes sense to keep it?

https://github.com/scala-steward-org/scala-steward/blob/4a42929bfae61393c3bc1ae3a5a98ae6a150d6f1/modules/core/src/main/scala/org/scalasteward/core/application/StewardAlg.scala#L109-L113